tests/auth/src/index.ts

```diff
@@ -1,62 +1,36 @@
 import { App } from '@microsoft/teams.apps';
 import { ConsoleLogger } from '@microsoft/teams.common/logging';
-import { MessageActivity } from '@microsoft/teams.api';
-import { Card, CodeBlock, Column, ColumnSet, TextBlock } from '@microsoft/teams.cards';
 import { DevtoolsPlugin } from '@microsoft/teams.dev';
 
 const app = new App({
   logger: new ConsoleLogger('@tests/auth', { level: 'debug' }),
   plugins: [new DevtoolsPlugin()],
 });
 
+// :snippet-start: auth-signout
 app.message('/signout', async ({ send, signout, isSignedIn }) => {
   if (!isSignedIn) return;
-  await signout();
+  await signout(); // call signout for your auth connection...
   await send('you have been signed out!');
 });
+// :snippet-end:
 
+// :snippet-start: auth-signin
 app.on('message', async ({ log, signin, api, isSignedIn }) => {
   if (!isSignedIn) {
-    await signin();
+    await signin(); // call signin for your auth connection...
     return;
   }
 
   const me = await api.user.me.get();
   log.info(`user "${me.displayName}" already signed in!`);
 });
 
-app.event('signin', async ({ send, api }) => {
+app.event('signin', async ({ send, api, token }) => {
   const me = await api.user.me.get();
-
-  await send(
```

Comment:
I kinda liked this result vs. the new one. Can we include the token in an optional showcard?

Reply:
I removed it because it would clutter the snippets. I could create a simpler card though.

Reply:
Fair! Atleast throw it in a console.log after.